.travis.yml

```diff
@@ -7,26 +7,26 @@ matrix:
   fast_finish: true
   include:
     - os: linux
-      node_js: '7'
-      env: WEBPACK_VERSION="2.2.0" JOB_PART=lint
+      node_js: '8'
+      env: WEBPACK_VERSION="2.6.0" JOB_PART=lint
     - os: linux
       node_js: '4.3'
-      env: WEBPACK_VERSION="2.2.0" JOB_PART=test
+      env: WEBPACK_VERSION="2.6.0" JOB_PART=test
     - os: linux
       node_js: '6'
-      env: WEBPACK_VERSION="2.2.0" JOB_PART=test
+      env: WEBPACK_VERSION="2.6.0" JOB_PART=test
     - os: linux
-      node_js: '7'
-      env: WEBPACK_VERSION="2.2.0" JOB_PART=coverage
+      node_js: '8'
+      env: WEBPACK_VERSION="2.6.0" JOB_PART=coverage
 before_install:
   - nvm --version
   - node --version
 before_script:
   - |-
     if [ "$WEBPACK_VERSION" ]; then
-      yarn add webpack@^$WEBPACK_VERSION
+      npm i -g webpack@^$WEBPACK_VERSION
```

Comment:
The behavior for `npm i` has changed in node 8. If we were going to use a mix of versions, global was the only common denominator.

Now that we are using npm@5 for everything, `npm i --no-save` is a viable option